.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Thank you for your contribution to Braintree. 
+
+> Before submitting this PR, note that we cannot accept language translation PRs. We support the same languages that are supported by PayPal, and have a dedicated localization team to provide the translations. If there is an error in a specific translation, you may open an issue and we will escalate it to the localization team.
```

Comment:
```suggestion
```

Reply:
I don't see any translation files in this repo so let's opt not to have this note

Reply:
Good catch. Thanks!